docs/docs/examples/model-inference/Stable-Diffusion-CKPT-Inference/index.md

```diff
@@ -101,119 +96,102 @@ Next, upload the image to the registry. This can be done by using the Docker hub
 docker push <hub-user>/<repo-name>:<tag>
 ```
 
-In our case
+So in our case, the command will look like this:
 
 ```bash
 docker push jsacex/stable-diffusion-ckpt
 ```
 
-After the repo image has been pushed to Docker Hub, you can now use the container for running on Bacalhau but before that you need to check whether your model is a ckpt file or not, if your model is a ckpt file you can skip to the running on Bacalhau if not the next section describes how to convert your model into the ckpt format.
+After the repo image has been pushed to Docker Hub, you can now use the container for running on Bacalhau. But before that you need to check whether your model is a `ckpt` file or not. If your model is a `ckpt` file you can skip to the running on Bacalhau, and if not - the next section describes how to convert your model into the `ckpt` format.
 
 
 ## Converting model weights to CKPT
 
 
 To download the convert script:
-```
+```bash
 wget -q https://github.com/TheLastBen/diffusers/raw/main/scripts/convert_diffusers_to_original_stable_diffusion.py
 ```
-To convert the model weights into CKPT format, the `--half` flag cuts the size of the output model from 4GB to 2GB:
+To convert the model weights into `ckpt` format, the `--half` flag cuts the size of the output model from 4GB to 2GB:
 
-```
-python convert_diffusers_to_original_stable_diffusion.py --model_path <path-to-the-model-weights>  --checkpoint_path <path-to-save-the-checkpoint>/model.ckpt --half
+```bash
+python3 convert_diffusers_to_original_stable_diffusion.py \
+    --model_path <path-to-the-model-weights>  \
+    --checkpoint_path <path-to-save-the-checkpoint>/model.ckpt \
+    --half
 ```
 
 ## Running a Bacalhau Job
 
-To do inference on your own checkpoint on Bacalhau you need to first upload it to your public storage, which can be mounted anywhere on your machine. In this case, we will be using [NFT.Storage](https://nft.storage/) (Recommended Option). To upload your dataset using [NFTup](https://nft.storage/docs/how-to/nftup/)  just drag and drop your directory it will upload it to IPFS
+To do inference on your own checkpoint on Bacalhau you need to first upload it to your public storage, which can be mounted anywhere on your machine. In this case, we will be using [NFT.Storage](https://nft.storage/) (Recommended Option). To upload your dataset using [NFTup](https://nft.storage/docs/how-to/nftup/) just drag and drop your directory it will upload it to IPFS.
```

Comment:
```suggestion
To do inference on your own checkpoint on Bacalhau you need to first upload it to your public storage, which can be mounted anywhere on your machine. In this case, we will be using [NFT.Storage](https://nft.storage/) (Recommended Option). To upload your dataset using [NFTup](https://nft.storage/docs/how-to/nftup/) drag and drop your directory and it will upload it to IPFS.
```